scripts/create-influx_debug-bucket.sh

```diff
@@ -3,3 +3,21 @@ set -e
 
 # creates the debug bucket after InfluxDB initialization
 influx bucket create --name "${INFLUX_DEBUG_BUCKET}" --org "${DOCKER_INFLUXDB_INIT_ORG}"
+
+# Creates the CAN_test bucket
+influx bucket create --name "CAN_test" --org "${DOCKER_INFLUXDB_INIT_ORG}"
+
+# Creates the GPS_test bucket
+influx bucket create --name "GPS_test" --org "${DOCKER_INFLUXDB_INIT_ORG}"
+
+# Creates the IMU_test bucket
+influx bucket create --name "IMU_test" --org "${DOCKER_INFLUXDB_INIT_ORG}"
+
+# Creates the CAN_prod bucket
+influx bucket create --name "CAN_prod" --org "${DOCKER_INFLUXDB_INIT_ORG}"
+
+# Creates the GPS_prod bucked
+influx bucket create --name "GPS_prof" --org "${DOCKER_INFLUXDB_INIT_ORG}"
```

Comment:
Fix typo

Reply:
Good catch. Should be GPS_prod.

Reply:
Fixed the typo thanks! **ALSO: I am not sure this script is running as expected?** When I do `sudo docker compose restart` I expected the buckets to be created on Influx but nothing seems to happen. I believe nothing happens because I never saw any errors with _prof being creating instead of _prod. In the end I just manually made the buckets on InfluxDB.